init.lua

```diff
@@ -475,6 +492,19 @@ local function register_torch_bomb(name, desc, dirs, min_range, blast_radius, te
 			if tnt_modpath then
 				tnt.boom(pos, {radius=blast_radius, damage_radius=blast_radius+3})
 			end
+			if mcl_explosions_modpath then
+			   mcl_explosions.explode(pos, blast_radius,
+						  {
```

Comment:
This table seems to be identical in the three calls that you've added. Please move the table to a local variable for re-use.

Reply:
changed